UnitTests/PaymentsCoreTests/AnalyticsEventRequest_Tests.swift

```diff
@@ -0,0 +1,66 @@
+import XCTest
+@testable import PaymentsCore
+
+// swiftlint:disable force_cast
+// swiftlint:disable implicitly_unwrapped_optional
+// swiftlint:disable force_try
+// swiftlint:disable force_unwrapping
```

Comment:
take it or leave it comment. I think these can be single lined like `// swiftlint:disable force_unwrapping implicitly_unwrapped_optional` if we wanted